src/assignments/11-higher-order-function.js

```diff
@@ -32,3 +33,10 @@ const users = [
 ];
 
 // Program 1: create a function and print the user details whose age>=30 (use filter and map)
+const getUserLess30 = (usersList) => usersList.filter(
+  (user) => user.age >= 30,
+).map(
+  (user, index) => `User ${index + 1}: ${user.name}, ${user.age} years old, from ${user.country}`,
+);
+
```

Comment:
to be simple, writing 1 function with filter and 1 with map to return a new array with users whose age over or = 30